packages/core/src/toolkit-provider/ToolkitProvider.tsx

```diff
@@ -80,10 +84,20 @@ const createThemedChildren = (
   }
 };
 
+interface ToolkitProviderThatAppliesClassesToBody {
+  children: ReactNode;
+  density?: Density;
+  theme?: ThemeNameType;
+  applyClassesToBody?: true;
```

Comment:
I think I would change these multiple boolean props to be a single enum prop as I don't think it makes sense to have both of these booleans as true e.g.

```
<ToolkitProvider
  applyClassesToBody={true}
  applyClassesToChild={true}
>
```

Instead with an enum you are only allowed one option:

```
<ToolkitProvider
  applyClassesTo="body"
>
```